index.js

```diff
@@ -72,9 +72,11 @@ var browserslist = function (selections, opts) {
         selections = selections.split(/,\s*/);
     }
 
-    if ( opts.stats || process.env.BROWSERSLIST_STATS ) {
+    var statsFile;
+
+    var getStats = function () {
```

Comment:
@AleshaOleg you can always change `complexity` settings, you you understand that they are wrong in this case.

Or move it to `var stats = browserslist.loadStats()`